### PR TITLE
add libc_wrappers

### PR DESCRIPTION
**Pull request by cdeepali**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Fixes https://github.com/open-ce/open-ce/issues/236.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: envs/tensorrt-env.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 packages:
 {% set py_ver = python | float %}
-{% if build_type == 'cuda' and py_ver < 3.8 %}
+{% if build_type == 'cuda' %}
```
Comment (npanpaliya) [inline envs/tensorrt-env.yaml:3]: Approved this if cpu builds work fine with this.

Comment (cdeepali) [reply envs/tensorrt-env.yaml:3]: Yes, cpu builds are fine.

Comment (bnemanich) [reply envs/tensorrt-env.yaml:3]: We might still need to add libc_wrappers to other environment files. Are there any environment files that include CuDNN, but not TensorRT?

Comment (cdeepali) [reply envs/tensorrt-env.yaml:3]: no change in cudnn repo for libc_wrappers.

Comment (jayfurmanek) [reply envs/tensorrt-env.yaml:3]: Currently there is no other env file that uses cudnn (and not TRT), but I suppose there could be. Or someone could create a custom one, in which case they'd have to know to put in the wrappers.

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**